src/main/resources/public/js/payoutMethods.js

```diff
@@ -130,6 +130,34 @@ $(document).ready(
                 );
             }
         );
+        $('#stripeDeleteButton').submit(
```

Comment:
@Lumi3011 Aici ar trebui ID-ul form-ului in care se afla butonul. Vezi mai jos :D